src/wazuh_modules/syscollector/tests/sysCollectorFlatbuffers/syscollectorFb_test.cpp

```diff
@@ -0,0 +1,286 @@
+/*
+ * Wazuh SyscollectorImp
+ * Copyright (C) 2015, Wazuh Inc.
+ * August 1, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "syscollectorFb_test.h"
+
+constexpr auto SYNC_HMSG_TEST_PATH {"sync_hmsg.mom"};
+constexpr auto SYNC_PMSG_TEST_PATH {"sync_pmsg.mom"};
+
+constexpr auto COMPONENT {"syscollector"};
+constexpr auto TYPE {"state"};
+
+void SyscollectorFbTest::SetUp() {};
+
+void SyscollectorFbTest::TearDown() {};
+
+TEST(SyscollectorFbTest, createBinaryHotfix)
```

Comment:
This test doesn't do any assertion. It only creates the file that is used in the test "readBinaryHotfix". So, this test could be merged with "readBinaryHotfix" in a unique test.